google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -182,6 +182,7 @@
     <project.converterProvider implementation="com.google.cloud.tools.intellij.appengine.migration.AppEngineStandardFacetConverterProvider"/>
     <project.converterProvider implementation="com.google.cloud.tools.intellij.appengine.migration.AppEngineStandardFacetMigrationConverterProvider"/>
     <framework.detector implementation="com.google.cloud.tools.intellij.appengine.facet.AppEngineFrameworkDetector"/>
+    <compileServer.plugin classpath="google-cloud-tools-jps-plugin.jar;appengine-plugins-core-0.1.14.jar;"/>
```

Comment:
Adding the tools lib to the class path this way means we'll have to manually change the jar name every time we bump the lib version, which is undesirable.
Any suggestions for improvement here? I think we can script some things in plugin.xml, but not custom stuff.